mini-e2e.groovy

```diff
@@ -0,0 +1,66 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/Yuggupta27/ceph-csi'
```

Comment:
Ideally you place a comment here, so that it is clear the URL/branch needs to change